holmes/plugins/skills/skill_loader.py

```
@@ -0,0 +1,214 @@
+            skill_path = Path(root) / SKILL_FILENAME
+            try:
+                skill = parse_skill_file(skill_path, source=source)
+                skills.append(skill)
+            except Exception as e:
+                logging.error(f"Failed to parse {skill_path}: {e}")
+
+    return skills
+
+
+def map_robusta_instruction_to_skill(
+    instr: "RobustaSkillInstruction",
```

Ruff (0.15.11)

[error] Undefined name `RobustaSkillInstruction`

(F821)